### ISSUE TITLE
What's the schedule?

### ISSUE THREAD

**Issue by clementi** [haskelllive/haskelllive#11]
How often will these episodes be?

**Comment by reinh** [comment]
Every week, hopefully released on Mondays!

**Comment by jamesthompson** [comment]
Bring on Ep 3!! We love this stuff!

**Comment by reinh** [comment]
Thanks! I know I haven't been keeping up with the schedule :( Life is sort of intervening. But I am going to try my best to get an episode out this week!

**Comment by ctbo** [comment]
I just wanted to let you know that I am also eagerly waiting for new episodes. Keep up the good work!

**Comment by J-Swift** [comment]
I'm still eagerly awaiting the next episode : ( The first two were really good...

**Comment by thomastjeffery** [comment]
Just so you know.. This is **too _awesome_** to leave hanging. You should make another episode.

**Comment by shinuza** [comment]
@reinh Are you OK?

**Comment by robertfall** [comment]
I am incredibly sad that there aren't more of these wonderful videos.

**Comment by wraithm** [comment]
I completely agree, these were fantastic!

**Comment by honza** [comment]
9 months since the last episode?

**Comment by reinh** [comment]
@honza I'm a terrible person. I want to make more but obviously I can't promise anything.

**Comment by honza** [comment]
@reinh No, not a terrible person. Quite the opposite.

**Comment by reinh** [comment]
@honza :heart: I appreciate the support and the prodding.

**Comment by apella-zz** [comment]
Just a small extra prod. You have a nice voice for video and a good presentation style. Any chance of you continuing this series, reinh?

**Comment by tbelaire** [comment]
I can't wait for HL:episode 3 to come out.
Wait a second... this almost feels like waiting for [Valve]([https://en.wikipedia.org/wiki/Half-Life_%28series%29])...
Jokes aside, I'm really looking forward to it.  In addition to the solid Haskell, you are very comfortable in vim and it's great to watch; I've learnt a few tricks from your videos.

**Comment by lb5tr** [comment]
Now I'm starting to understand whole excitement about Haskell, theses videos are remarkable, very well presented and clear (demanding some _basic_ Haskell background, though).
By the way, I'm really admiring your vim-skills. Can't wait for next episodes! Keep it up!

**Comment by kippr** [comment]
Really enjoyed the first two episodes, thanks! Really useful way for an experienced programmer to get an introduction.

Don't feel bad about slipping, you set yourself quite an ambitious goal! As coders, we all know how hard it can be to stick to committed estimates, and this isn't even your day job! ;)

**Comment by houshuang** [comment]
Just to throw another vote in, these are really good videos - I actually used a few of the tips I picked up on exercism.io exercises. Would really love to see more videos!

**Comment by majjoha** [comment]
I agree with all the others in here. The first two episodes are so great. I look truly forward to the third, @reinh.

**Comment by sbditto85** [comment]
Just a +1 for more videos, great job!

**Comment by lpierron** [comment]
Just a +1 for more videos, very great job!

**Comment by wutka** [comment]
Thanks for the work you did here, you did a great job. While I appreciate some of the Haskell tips I picked up here, I am most grateful for seeing your development setup and how well you use it.

**Comment by danielpetri1** [comment]
Still eagerly waiting for more episodes 11 years later:)
